gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -916,7 +916,7 @@ public class ConfigurationKeys {
    * Kafka job configurations.
    */
   public static final String KAFKA_BROKERS = "kafka.brokers";
-  public static final String KAFKA_BROKER_TO_SIMPLE_NAME_MAP_KEY = ConfigurationKeys.KAFKA_BROKERS + ".simpleNameMap";
+  public static final String KAFKA_BROKER_TO_SIMPLE_NAME_MAP_KEY = ConfigurationKeys.KAFKA_BROKERS + "ToSimpleNameMap";
```

Comment:
this definition is a little wired. Can you just change it to be "kafka.brokersToSimpleNameMap"